documentation/SA1024.md

```diff
@@ -45,6 +45,11 @@ switch (x)
 }
 ```
 
+A colon that appears as part of format string component must not have leading whitespace characters. For example:
```

Comment:
I would prefer `a string interpolation formatting component` instead of `format string component`